src/Lean/Server/Requests.lean

```diff
@@ -49,6 +49,7 @@ end RequestError
 structure RequestContext where
   srcSearchPath : SearchPath
   docRef        : IO.Ref FileWorker.EditableDocument
+  hLog          : IO.FS.Stream
```

Comment:
I usually just use `dbg_trace` in request handlers fwiw, works well enough

Reply:
Oh, at some past point I had issues with that since the main thread's `stderr` wasn't inherited by task threads. It seems to work now, but I see no harm in leaving that log stream field.